packages/components/accordion/__tests__/accordion.test.tsx

```diff
@@ -30,7 +30,7 @@ describe("Accordion", () => {
 
     expect(() => wrapper.unmount()).not.toThrow();
 
-    expect(spy).toBeCalledTimes(0);
+    expect(spy).toHaveBeenCalledTimes(0);
```

Comment:
`toBeCalledTimes` is deprecated.
see: https://github.com/jestjs/jest/issues/13164